service/git/syncer.go

```diff
@@ -53,8 +54,15 @@ func (s *Syncer) Sync(ctx context.Context,
 	return s.fetch(ctx, frp.InternalRepositoryURL, r, refspecs)
 }
 
-func (s *Syncer) fetch(ctx context.Context, repoURL string, r *git.Repository, refspecs []config.RefSpec) error {
+func (s *Syncer) fetch(ctx context.Context, repoURL string, r *git.Repository, refspecs []config.RefSpec) (err error) {
 	log.Infof("fetching references for repository %s: %v", repoURL, refspecs)
+	start := time.Now()
+	defer func() {
+		if err == nil {
+			log.Debugf("references %v fetched for repository %s in %v", repoURL, refspecs, time.Now().Sub(start))
```

Comment:
should we use log with field `duration`, to follow the [guide conventions](https://github.com/src-d/guide/blob/master/engineering/conventions.md#logging) more closely?

Reply:
I think for debug logs it doesn't make much sense. Debug level shouldn't be enabled in production. Though I'm changing it just for the sake of consistency with other log messages.